include-fragment-element.js

```diff
@@ -65,6 +65,18 @@ export default class IncludeFragmentElement extends HTMLElement {
     }
   }
 
+  get accept() {
+    return this.getAttribute('accept')
+  }
+
+  set accept(val) {
+    if (val) {
+      this.setAttribute('accept', val)
+    } else {
+      this.removeAttribute('accept')
```

Comment:
I think we should only remove boolean attributes when given falsey values. Is there precedent in other elements for removing non-boolean attributes?

Reply:
I don't believe so. This was the case for `src` too, so I updated to remove both `removeAttribute` calls from `setter`.